src/main/java/Duke.java

```diff
@@ -1,10 +1,342 @@
+import duke.command.DukeException;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.Scanner;
+
 public class Duke {
+    /**
+     * Executes the "Duke" program.
+     *
+     * @param args The command line arguments.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //Initialisation
+        String line = "start";
+        Task[] list = new Task[100];
+        int listSize = 0;
+        Scanner input = new Scanner(System.in);
+        System.out.println("Hello! I'm Duke");
```

Comment:
Try to make string literals FINAL and try to define string literals before using them